modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/pages/Testflow/modal/index.tsx

```diff
@@ -40,7 +41,16 @@ const TestflowAssignUserModal: React.FC<TestflowAssignUserModalProps> = ({
 			last={
 				<Form.Footer
 					onClose={onClose}
-					onSubmit={() => onSave(state)}
+					onSubmit={() =>
+						state.length
+							? onSave(state)
+							: Liferay.Util.openToast({
+									message: i18n.translate(
+										'mark-at-least-one-user-for-assignment'
+									),
+									type: 'danger',
+							  })
```

Comment:
I am not sure if this validation should be here, probably where it's called, during the onSave according to my last comment